src/v2/Apps/Order/Components/ItemReview.tsx

```diff
@@ -74,14 +62,19 @@ export const ItemReview: React.SFC<ItemReviewProps> = ({
         artworkDimensions &&
         dimensionsDisplay(artworkDimensions)}
       {attribution_class && (
-        <Serif size="2" color="black60">
+        <Text variant="sm" color="black60">
           {attribution_class.shortDescription}
-        </Serif>
+        </Text>
       )}
     </Flex>
-    <ImageBox>
-      <img alt={`${title} by ${artist_names}`} src={url} />
-    </ImageBox>
+    <Box maxWidth={185}>
+      <Image
+        maxHeight={375}
+        width="100%"
+        src={url}
+        alt="Please make sure you describe the image"
```

Comment:
Should either describe this image of relevant or just give it `alt=""` if it should be ignored by screen readers.

Reply:
Also this is sort of weird; just give it a width and height, get rid of the outer box and the max-width/height?

Reply:
make sense!